addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -59,6 +59,8 @@ def _holiday_status_id_domain(self):
     date_to = fields.Datetime(
         'End Date', readonly=True, copy=False,
         states={'draft': [('readonly', False)], 'confirm': [('readonly', False)]}, tracking=True)
+    display_date_from = fields.Date('Start Date', compute='_compute_date_from', inverse='_inverse_date_from', store=False)
+    display_date_to = fields.Date('End Date', compute='_compute_date_to', inverse='_inverse_date_to', store=False)
```

Comment:
Labels for both new fields should be different than labels for existing fields: `date_from` and `date_to`